src/runtime/GlobalObjectBuiltinArray.cpp

```diff
@@ -1489,6 +1489,22 @@ static Value builtinArrayShift(ExecutionState& state, Value thisValue, size_t ar
     return first;
 }
 
+static Value builtinArrayUnscopables(ExecutionState& state, Value thisValue, size_t argc, Value* argv, bool isNewExpression)
+{
+  Value blacklist = state.context()->globalObject()->objectCreate()->call(state, Value(), 1, &blacklist);
+  blacklist.asObject()->defineOwnPropertyThrowsException(state, ObjectPropertyName(state.context()->staticStrings().copyWithin), ObjectPropertyDescriptor(Value(true)));
```

Comment:
I think you can use `defineOwnProperty` method here, since `blackList` is an intact object, so it cannot throw any exception during the property set operation.